docs/en/plugins/owners.md

```diff
@@ -48,23 +48,27 @@ Note: Because maintainers are not attached to any SIG, they will be fetched dire
 | default_sig_name          | string                  | Set the default SIG for this repository                                                                                                                              |
 | default_require_lgtm      | int                     | Set the default number of lgtm required for this repository                                                                                                          |
 | require_lgtm_label_prefix | string                  | The plugin supports specifying the number of lgtm required for the current PR by label, and this option is used to set the prefix of the relevant label              |
 | trusted_teams             | []string                | List of trusted GitHub team names (generally maintainers team)                                                                                                       |
+| committer_teams           | []string                | Specify a list of GitHub Team names whose members can serve as Committers                                                                                            |
+| reviewer_teams            | []string                | Specify a list of GitHub Team names whose members can serve as Reviewers                                                                                             |
 | use_github_permission     | bool                    | Use GitHub permissions                                                                                                                                               |
 | branches                  | map[string]BranchConfig | Branch granularity parameters configuration, map structure key is the branch name, the configuration of the branch will override the configuration of the repository |
 
 ### BranchConfig
 
-| Parameter Name        | Type     | Description                                            |
-| --------------------- | -------- | ------------------------------------------------------ |
-| default_require_lgtm  | int      | Set the default number of lgtm required for the branch |
-| trusted_teams         | []string | Set up a trusted GitHub team for the branch            |
-| use_github_permission | bool     | Use GitHub permissions                                 |
+| Parameter Name        | Type     | Description                                                                             |
+|-----------------------|----------|-----------------------------------------------------------------------------------------|
+| default_require_lgtm  | int      | Set the default number of lgtm required for the branch                                  |
+| trusted_teams         | []string | Set up a trusted GitHub team for the branch                                             |
+| committer_teams       | []string | Set up a list of GitHub Team names for the branch whose members can serve as Committers |
```

Comment:
If they are the same effectively, I'd prefer just use trusted_teams or rename trusted_teams to committer_teams if you like it.

Reply:
My idea is that we can now add three options to add a new authentication mode: `use_github_teams`、`committer_teams`、`reviewer_teams`,  because I don't want to mix committers and reviewers from sig member API and GitHub teams.

- When `use_github_teams` is enabled, we get the list of committers and reviewers through the specified Github team.

- When `use_github_teams` is not enabled, we obtain committers and reviewers through the sig member API provided by ti-community-bot. In this mode, it is allowed to add some additional members through the trusted team's option.

**But** I think it may be acceptable to temporarily merge committers and reviewers from the two sources, and we will remove the sig member API part later. What do you think?

Reply:
I get your point. Then it is fine we keep trusted_teams for case "When use_github_teams is not enabled," and later clean up logics when the authentication changes successfully and stably.